R/module_nested_tabs.R

```diff
@@ -314,12 +314,14 @@ srv_nested_tabs.teal_module <- function(id, datasets, modules, is_module_specifi
     data,
     eventReactive(
       trigger_data(),
-      c(
-        get_rcode_str_install(),
-        get_rcode_libraries(),
-        get_datasets_code(datanames, datasets, hashes),
-        teal.slice::get_filter_expr(datasets, datanames)
-      )
+      {
+        c(
+          get_rcode_str_install(),
+          get_rcode_libraries(),
+          get_datasets_code(datanames, datasets, hashes),
+          teal.slice::get_filter_expr(datasets, datanames)
+        )
+      }
```

Comment:
Since this is a single call the braces are not required. If this is a style choice, ok.